lib/api.ts

```diff
@@ -98,6 +98,25 @@ export async function getContributorBySlug(file: string, detail = false) {
   }
 
   const slackMessages = await getSlackMessages(data.slack);
+  const UtctoDate = (date: number | string) => {
+    let dateObject;
+
+    if (typeof date == undefined) {
+      return undefined;
+    }
+    if (typeof date === "number") {
+      let timestamp = Number(date) * 1000;
+      dateObject = new Date(timestamp);
+      return dateObject.toISOString();
+    } else if (typeof date === "string") {
+      dateObject = new Date(date);
+
+      return dateObject.toISOString();
+    }
+  };
+
+  let Firstactivity =
+    activityData?.activity[activityData.activity.length - 1]?.time;
```

Comment:
https://google.github.io/styleguide/jsguide.html#naming-non-constant-field-names

```suggestion
  const firstActivity =
    activityData?.activity[activityData.activity.length - 1]?.time;
```